### PR TITLE
Add all system properties to args for dev mode.

### PR DESCRIPTION
**Pull request by soul2zimate**
User input VM argument is ignored in development mode because it only takes argument starts with **quarkus.**

### CODE REVIEW

**Comment by stalep** [comment]
Looks reasonable, but we should also make the same change to QuarkusDev for Gradle as well.

File: devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java
Comment (Sanne) [inline devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java]: Please see comment on line 226: not sure if this is a good idea?

But if we decide to apply the patch, the comment should be updated.

**Review by gsmet — CHANGES_REQUESTED** [review]
Please don't merge it now. We need to discuss this and we are about to release a new version.

Marking it as "Request changes" so it's not merged blindly.

**Comment by Sanne** [comment]
Thanks @gsmet , was about to say the same.

We need to find a good balance here, I can see @soul2zimate 's point: it should be really easy to pass such properties transparently, and possibly even automatically as people shouldn't be bothered by this, and be able to expect this to just work.

Maybe by inspecting the properties which were set on the spwaning JVM command line?

**Comment by gsmet** [comment]
Yes, I agree we need to find a solution and the status quo is not satisfying.

@dmlloyd @stuartwdouglas pretty sure you will have some interesting insights on this one.

**Comment by geoand** [comment]
I think this is now out of date after the latest changes about dev mode merged to master, right?

**Comment by stalep** [comment]
yes, this should be fixed now.

**Pull request closed without merging**
